contracts/package.json

```diff
@@ -33,7 +33,7 @@
     "ethereum-waffle": "^3.2.0",
     "ethers": "^5.0.17",
     "fs-extra": "^9.0.1",
-    "hardhat": "^2.6.0",
+    "hardhat": "^2.6.8",
```

Comment:
(arbitrary code location to allow for threading)

If the user does not already have Polygon configured in their wallet, attempting to switch to it seems to fail silently. I think most apps will ask to add the network for the user if it doesn't exist. Can we easily do this as well?

![umbra-polygon-no-network-switch](https://user-images.githubusercontent.com/1479802/145287104-4de364d3-8781-4094-9046-6b6e1303f4c8.gif)

Reply:
We had this feature, and I accidentally broke it when adding the network selector. Details + fix in 1203fca